model_cdr_cdr_token.go

```diff
@@ -0,0 +1,225 @@
+/*
+OCPI CDRs module
+
+Specification for OCPIs CDRs handlers
+
+API version: 2.2.1
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package OCPI
+
+import (
+	"encoding/json"
+)
+
+// checks if the CdrCdrToken type satisfies the MappedNullable interface at compile time
+var _ MappedNullable = &CdrCdrToken{}
+
+// CdrCdrToken struct for CdrCdrToken
+type CdrCdrToken struct {
```

Comment:
Naming here isn't optimal

Reply:
resolved as cdrBody, let me know if you have would have better idea 🤔